### ISSUE TITLE
Task379 - Add business logic to return guided experience resource to the '/search' endpoint

### ISSUE THREAD

**Issue by aroubin** [LSCOIT/LegalNavigator#300]
_From @vennasr on August 20, 2018 11:6_

Updated the business logic to return guided assistant resource and updated the response in angular component as well.

_Copied from original issue: Microsoft/Access2Justice/pull/387_

**Comment by aroubin** [comment]
_From @vennasr on August 21, 2018 15:28_

Winnie, I have modified the logic now so all the search results you will get in the search response. I have updated the search code to return guided assistant id along with all the results which were getting previously. 
Implemented the guided assistant side bar component to bind the proper guided assistant id. Whenever the id is not present then it will take the user to the guided assistant search page.
Today I have added one more curated experience to test the functionality with the similar data(Just updated the text from A2J to **Access to Justice** in curated experience) but mapped to different topic id. 
Search term: 
1) Divorce 
2) Child abuse
